src/urlConnections.js

```diff
@@ -1,4 +1,5 @@
 const URLS = {
   BACKEND_URL: process.env.BACKEND_URL,
 };
+
 export default URLS;
```

Comment:
It seems you have two files with the same data consider removing one of them.
src/urlConnections.js